Lekce_7_HW/Book.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace Lekce_7_HW
+{
+    internal class Book
+    {
+        private string _title;
+        private string _author;
+        private int _numberOfPages;
+        private int _currentPage = 0;
+        
+        
+
+        public Book(string title, string author, int numberOfPages)
+        {
+            _title = title;
+            _author = author;
+            _numberOfPages = numberOfPages;
+        }
+
+        public void Read(int pages)
+        {
+        _currentPage += pages;
```

Comment:
Osetreni varuje uzivatele, ze precetl neocekavany pocet stran, ale uz to neresi realitu, ze objekt pak zustane v nevalidnim stavu.

Priklad:
Mam objekt `medaPu` typu Book, ktera ma 10 stran. Ja na ni zavolam
`medaPu.Read(666);`
Dostanu hlasku, ze jsem to trochu prepisknul, ale pak muzu hezky dal volat
`medaPu.Display()`
kolikrat chci a budu dostavat nesmyslny vypis.

imo bych v takovehle situaci hodnotu currentPage vubec nemenil a jen zahlasil jako nevalidni vstup a nebo ji nechal na maximalni validni hodnote. Kterou volbu vyberes necham na tobe :)